README.md

```diff
@@ -4,6 +4,14 @@
 
 The Islandora Compound Object Solution Pack enables generic parent-child relationships between objects. The object view of a compound object is replaced by the view of its first child object. The included "Islandora Compound Object Navigation" block provides a thumbnail navigation of an object's siblings. A "Compound" management tab allows for the addition and removal of parent and child objects for each object.
 
+## Requirements
+
+This module requires the following modules/libraries:
+
+* [Islandora](https://github.com/islandora/islandora)
+* [JAIL](https://github.com/sebarmeli/JAIL) library 
```

Comment:
Forgot about this for the committers call, but I guess any other @Islandora/7-x-1-x-committers could help with this. JAIL  is documented as a requirement but the module itself does not impose it as a requirement (only needed if enabled in the block). I personally vote for consistency across our docs but would also make sure people don't have to install libraries they won't be using. Any thoughts?